src/org/hammerhead226/sharkmacro/Parser.java

```diff
@@ -238,4 +277,12 @@ private static boolean touchDirectory(String directory) {
 		return true;
 	}
 
+	private static String format(String directory, String filename) {
```

Comment:
You probably meant to doc this.

Good call on a new method, though.

Reply:
Oops, you right.